### DISCUSSION TITLE
Customizing icons

### DISCUSSION THREAD

**Discussion by andresatierf** [LazyVim/LazyVim#946] [Q&A] [answered]
Is it possible to change the icons present in `lazyvim/config/init.lua`?
I tried to do it by overriding the imported spec:
```lua
require("lazy").setup({
  spec = {
    -- add LazyVim and import its plugins
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
   },
  ...
}
```
like this:
```lua
return {
  {
    "LazyVim/LazyVim",
    opts = {
      icons = {
        git = {
          added = " ▏",
          modified = " ▏",
          removed = " ▏",
        },
      },
    },
  }
}
```
and like this:
```lua
return {
  {
    "lazyvim.plugins",
    opts = {
      icons = {
        git = {
          added = " ▏",
          modified = " ▏",
          removed = " ▏",
        },
      },
    },
  }
}
```
but it didn't seem to do anything.
Am I missing something or is there no way to override these icons?

**Comment by jyuan0** [comment]
Maybe it's where you're putting the spec? The second spec with `LazyVim/LazyVim` works for me in `plugins/core.lua`.
```
return {
  {
    "LazyVim/LazyVim",
    opts = {
      icons = {
        git = {
          added = " ▏",
          modified = " ▏",
          removed = " ▏",
        },
      },
    },
  }
}
```

**Reply by andresatierf** [reply]
I have a file inside `plugins/` with the following contents:
```lua
return {
  {
    "LazyVim/LazyVim",
    opts = {
      defaults = {
        autocmds = true, -- lazyvim.config.autocmds
        keymaps = false, -- lazyvim.config.keymaps
      },
      icons = {
        git = {
          added = " ▏",
          modified = " ▏",
          removed = " ▏",
        },
      },
    },
  },
}
```
the `keymaps = false` seems to work but not the git icons

**Reply by jyuan0** [reply]
Both are working for me (disabling the keymaps and changing the icons). Sorry, no idea why this isn't working for you.

**Reply by jyuan0** [reply]
It's a wild guess and a shot in the dark, but maybe this is a font issue?

**Reply by andresatierf** [reply]
I tried to replace those symbols with regular character and still didn't help

**Reply by andresatierf** [reply]
I just now realized that this table changes the icons in the status line, when I was trying to change the icons in the statuscolumn :facepalm:. Thanks for your help, that is indeed working

**Comment by andresatierf** [comment]
@folke is there any incentive to adding the 'gutter' or 'statuscolumn' icons to this table to make it easy to customize? Currently, they are defined in the gitsigns spec

**Reply by folke** [reply, marked as answer]
Those are specific to gitsigns, so it makes more sense to leave them there.
You can easily override those in the gitsigns opts.

**Reply by andresatierf** [reply]
Thanks
